translation/source/site.xml

```diff
@@ -990,4 +989,12 @@ Leave empty to start games from the normal initial position.</string>
   <string name="showMeEverything">Show me everything</string>
   <string name="lichessPatronInfo">Lichess is a charity and entirely free/libre open source software.
 All operating costs, development, and content are funded solely by user donations.</string>
+  <string name="created">Created</string>
+  <string name="recentlyPlayed">Recently played</string>
+  <string name="bestResults">Best results</string>
+  <string name="stats">Stats</string>
+  <string name="xTournamentStats">%s tournament stats</string>
+  <string name="rankAvgHelp">The rank average is a percentage of your ranking. Lower is better.
+
+For instance, being ranked 3 in a tournament of 100 players = 3%%. Being ranked 10 in a tournament of 1000 players = 1%%.</string>
```

Comment:
Do these strings need to be converted to arena.xml?

Reply:
Are they about arena tournaments? If so, then yes.